frontend/src/components/Auth/Logout.js

```diff
@@ -10,7 +13,7 @@ const Logout = () => {
 
   return (
     <div>
-      <input type="button" value="Logout" onClick={handleClick} />
+      {logoutIcon} <input type="button" value="Logout" onClick={handleClick} />
```

Comment:
Can consider making logoutIcon clickable with the Logout button - wrap it inside button tag or input tag along with 'Log Out'